src/sentry/notifications/helpers.py

```diff
@@ -255,6 +255,9 @@ def get_scope(
     Figure out the scope from parameters and return it as a tuple.
     TODO(mgaeta): Make sure the user/team is in the project/organization.
     """
+    if team:
```

Comment:
This needs to be below the project and organization checks. Otherwise a project-specific team notification setting's scope will be `TEAM` instead of `PROJECT`.

Reply:
We agreed to not have any project specific team notification settings though

Reply:
I guess it works fine the other way too